react/features/e2ee/components/E2EESection.js

```diff
@@ -142,10 +153,23 @@ class E2EESection extends Component<Props, State> {
  * @returns {Props}
  */
 function mapStateToProps(state) {
-    const { enabled } = state['features/e2ee'];
+    const { enabled: toggled, maxMode } = state['features/e2ee'];
```

Comment:
Pleasse name this e2eeEnabled or something, otherwise it's easy to confuse it with the state of the switch.